src/styles/block/stopwatch.scss

```diff
@@ -0,0 +1,50 @@
+.stopwatch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+
+  &__border {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: 200px;
+    height: 200px;
+    border: 3px blue dotted;
+    border-radius: 50%;
+  }
+
+  &__hand {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+
+    &--minute {
+      height: 100px;
+      width: 2px;
+      transform-origin: top;
+      translate: -1px;
```

Comment:
You can use another way to positioning this element. Because right now you don't follow checklist. [checklist par. 3](https://github.com/mate-academy/layout_stop-watch/blob/master/checklist.md)

Maybe try to make this transform more adaptive? Like, no matter what width I set, the element was always positioned in the center. Think, how we can do this with transform property.